app/services/message_service.rb

```diff
@@ -0,0 +1,29 @@
+class MessageService
+  attr_accessor :text, :conversation, :user
+
+  def initialize(text, conversation, user)
```

Comment:
I think it's better just set the conversation here, and the `create_message` method, add the text and user:
```ruby
def initialize(conversation)
...
end

def create_message(user, text)
...
end
```

Then `MessageService.new(conversation).create_message(user, text)`